batchflow/models/torch/resnet.py

```diff
@@ -78,7 +78,7 @@ def default_config(cls):
                                         pool_size=3, pool_strides=2)
 
         config['body/encoder/num_stages'] = 4
-        config['body/encoder/order'] = ['block']
+        config['body/encoder/order'] = ['skip', 'block']
```

Comment:
Can't understand, why do you need this skip. It seems to me, that the skip is discarded in `EncoderModule.forward` because `return_all=False` is passed while creating EncoderModule in  `Encoder.body`. `return_all` isn't set in config for ResNet

Reply:
It can be useful if one uses `ResNet18`, for example, as `base_model` for encoder-decoder.